pyodide-build/pyodide_build/buildall.py

```diff
@@ -261,13 +262,14 @@ def build_packages(packages_dir: Path, outputdir: Path, args) -> None:
         if pkg.library:
             continue
         if pkg.shared_library:
-            package_data["shared_library"][name] = True
-        package_data["dependencies"][name] = [
-            x for x in pkg.dependencies if x not in libraries
+            package_data["shared_library"][name.lower()] = True
+        package_data["dependencies"][name.lower()] = [
+            x.lower() for x in pkg.dependencies if x not in libraries
         ]
-        package_data["versions"][name] = pkg.version
+        package_data["versions"][name.lower()] = pkg.version
         for imp in pkg.meta.get("test", {}).get("imports", [name]):
-            package_data["import_name_to_package_name"][imp] = name
+            package_data["import_name_to_package_name"][imp] = name.lower()
+        package_data["orig_case"][name.lower()] = name
```

Comment:
Wait why do we care about the original case, here? Using all lowercase would be enough, wouldn't it?

Reply:
We need the original case to figure out what the right file path is for the http request.